homeassistant/components/google_cloud/tts.py

```diff
@@ -172,12 +172,10 @@ async def _async_get_tts_audio(
             _LOGGER.error("Error: %s when validating options: %s", err, options)
             return None, None
 
-        encoding: texttospeech.AudioEncoding = texttospeech.AudioEncoding[
-            options[CONF_ENCODING]
-        ]  # type: ignore[misc]
-        gender: texttospeech.SsmlVoiceGender | None = texttospeech.SsmlVoiceGender[
+        encoding = texttospeech.AudioEncoding(options[CONF_ENCODING])
+        gender: texttospeech.SsmlVoiceGender | None = texttospeech.SsmlVoiceGender(
```

Comment:
could it still be None then?

Reply:
`if voice` is given, it's set to `None`. So the explicit annotation is necessary here.
`encoding` will never be set to `None` though.